tensorflow/core/platform/hadoop/hadoop_file_system.cc

```diff
@@ -170,6 +172,12 @@ Status HadoopFileSystem::Connect(StringPiece fname, hdfsFS* fs) {
   if (ticket_cache_path != nullptr) {
     hdfs_->hdfsBuilderSetKerbTicketCachePath(builder, ticket_cache_path);
   }
+
+  const char* hdfs_user = getenv("HDFS_USER");
```

Comment:
HDFS_USER -> HADOOP_USER_NAME to match existing HDFS conventions?

Reply:
Yes you are right. I'll update it.